### PR TITLE
Split CWS publish article

### PR DESCRIPTION
**Pull request by AmySteam**
Fixes #https://github.com/GoogleChromeLabs/Extension-Docs/issues/219

Changes proposed in this pull request:

-
-
-

### CODE REVIEW

File: site/en/docs/webstore/set-up-account/index.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+{% Img src="image/SHhb2PDKzXTggPGAYpv8JgR81pX2/KZbN4VNG8mEFEDsiWglX.png", alt="Contact email field
+showing as unverified", width="250", height="98", class="screenshot screenshot--filled" %}
+
+Click the **Verify email** link to send the verification link to your email.
+
+## Next steps
+
+- [Prepare your extension][prepare]. 
+- [Publish your extension][publish].
```
Comment (jpmedley) [inline site/en/docs/webstore/set-up-account/index.md:55]: This link is broken.

**Review by jpmedley — CHANGES_REQUESTED** [review]
Aside from the broken link I already noted, the [next step items on register](https://developer.chrome.com/docs/extensions/mv3/user_interface/#localize) don't match the actual next steps.

File: site/en/docs/webstore/set-up-account/index.md (hunk 1 above)
Comment (AmySteam) [reply site/en/docs/webstore/set-up-account/index.md:55]: Good catch! Fixed links 👍

**Comment by AmySteam** [comment]
@jpmedley PTAL

**Review by jpmedley — APPROVED** [review]

**Pull request merged**